docs/updating_dataset.md

```diff
@@ -23,11 +25,20 @@ OR
 
 _When in doubt, rerun both CLI commands._
 
-## If only the subjects have changed
+### If only the subjects have changed
 If subjects have been added to or removed from the dataset but the phenotypic TSV is otherwise unchanged (i.e., only new or removed rows, without changes to the available variables), you will need to:
 
 - **Generate a new graph-ready data file** for the dataset by [re-running the CLI](cli.md) (`pheno` and `bids` steps) on your updated TSV and existing data dictionary
 
+## Following a change in the _Neurobagel data model_
+
+As Neurobagel refines the way that specific subject properties are modeled as graph data, new tool releases will occasionally introduce breaking changes to the data model for subject-level information in a `.jsonld` graph data file.
```

Comment:
```suggestion
As Neurobagel continues developing the data model, new tool releases may introduce breaking changes to the data model for subject-level information in a `.jsonld` graph data file. This will be highlighted in the release notes.
```